content/TrueCommand/Clustering/ManageClusters.md

```diff
@@ -118,6 +122,8 @@ Click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> ic
 
 Check the **Confirm** box, then click **OK** to delete the cluster.
 
+On the Dashboard, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```

Comment:
same comment as above. As this repeats, is this a good candidate for a snippet to maintain the content in one place?
```suggestion
On the **Dashboard**, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> icon and select **Services**. Stop the Gluster service and uncheck **START ON BOOT**.
```